src/DataDisplay.tsx

```diff
@@ -128,14 +129,28 @@ const DataDisplay: React.FC<DataDisplayProps> = ({
         )}
       </div>
       <div className="cyto-below">
-        <select onChange={(e) => setLayout(e.target.value)} value={layout}>
-          <option value="random">random</option>
-          <option value="grid">grid</option>
-          <option value="circle">circle</option>
-          <option value="cose">cose</option>
-          <option value="concentric">concentric</option>
-          <option value="breadthfirst">breadthfirst</option>
-        </select>
+        <FormControl variant="outlined" className="map-style">
+          <InputLabel id="demo-simple-select-outlined-label">
```

Comment:
I believe this ID could be something else

Reply:
lol sorry copy and paste from material-ui. I actually did this one in like 10 minutes. 

![2a77f19121f1994aff17008a4811b3e0](https://user-images.githubusercontent.com/2829924/95682682-9c588d80-0bb4-11eb-8978-61eb345d8cc5.png)